Guide/tailwindcss.markdown

```diff
@@ -97,8 +97,13 @@ We also need a CSS entry point for Tailwind. Create a new file at `tailwind/app.
 We need to add a new build command for starting a tailwind build process to our `Makefile`. For that append this to the `Makefile` in your project:
 
 ```makefile
-tailwind-dev:
-	tailwindcss -c tailwind/tailwind.config.js -i ./tailwind/app.css -o static/app.css --watch
+# Install npm packages.
+node_modules:
```

Comment:
![image](https://github.com/digitallyinduced/ihp/assets/125707/11a0d8a1-cda5-4e02-b8b0-f9be7e8d24e8)

Reply:
Likely because you already have a node_modules directory. Can you delete it and try again?

Reply:
I have tried again (using [this PR](https://github.com/amitaibu/ihp-landing-page/pull/34)), but the same message

![image](https://github.com/digitallyinduced/ihp/assets/125707/97dc34a8-95bd-4bed-b3ad-a84efc36f631)